src/periodic.jl

```diff
@@ -0,0 +1,85 @@
+const PeriodicFunction = Union{typeof(cos), typeof(sin)}
```

Comment:
[blue style](https://github.com/invenia/BlueStyle#global-variables) recommends capitalising these.

But I'm personally indifferent in this case since it's a type.

Reply:
Hmm, yeah I prefer camel case for types so it's easier to tell visually.